SirinesCall/setup-SirinesCall.tp2

```diff
@@ -236,7 +236,7 @@ LANGUAGE ~D
          ~sirinescall/translations/russian/setup-sirinescall.tra~
 LANGUAGE ~Simplified Chinese~
          ~schinese~
-         ~sirinescall/translations/schinese/setup-sirinescall.TRA~
```

Comment:
You need to load setup-sirinescall.tra first, to properly load the content for both BGT and EE. Probably installation will fail as you are not loading the characters name. After loading setup simply add loadinfg the install.tra file.

Reply:
Thank you,fixed now.